focus-ios/Blockzilla/Onboarding/OnboardingFactory.swift

```diff
@@ -185,6 +185,11 @@ fileprivate extension String {
         value: "Agree and Continue",
         comment: "Label for the agreement button in the Terms of Use screen."
     )
+    static let termsOfUseButtonTextV2 = NSLocalizedString(
```

Comment:
Hi @razvanlitianu I'm not sure how the strings are handled in Focus, I've follow the process we have in Firefox to create a new version but let me know if the process is different